lib/commands/screenshot.js

```diff
@@ -0,0 +1,19 @@
+import { getScreenshot } from 'node-simctl';
+
+
+let commands = {}, helpers = {}, extensions = {};
+
+commands.getScreenshot = async function () {
+  let screenshot;
+
+  try {
+    screenshot = await getScreenshot(this.opts.udid);
+  } catch (e) {
+    return this.proxyCommand('/screenshot', '/GET');
```

Comment:
Also, `return await`, or, to make it of the same sort as the previous call, `screenshot = await this.proxyCommand('/screenshot', '/GET');`.